packages/app/src/specs/banners/ConnectProjectBanner.vue

```diff
@@ -55,19 +55,27 @@ query ConnectProjectBanner {
 }
 `
 
-withDefaults(defineProps<{
+const props = withDefaults(defineProps<{
   modelValue: boolean
   hasBannerBeenShown: boolean
+  bodyCopyOptions?: CohortOption[]
 }>(), {
   modelValue: false,
   hasBannerBeenShown: true,
+  bodyCopyOptions: () => {
+    return [
+      { cohort: 'A', value: 'specPage.banners.connectProject.contentA' },
+      { cohort: 'B', value: 'specPage.banners.connectProject.contentB' },
+    ]
+  },
```

Comment:
Yes, the prop was just for testablity because of the randomness.  I will try out injecting the algorithm instead.  I think I see where you are going with that.

Reply:
Left this alone for now, but can look at a refactor if this gets touched again.